### PR TITLE
API-15366 Added encounter.read to missing scope.

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**

### CODE REVIEW

File: data-query-tests/src/test/java/gov/va/api/health/dataquery/tests/dstu2/JoshBot.java [hunk 1]
```diff
@@ -69,6 +69,7 @@ public static void main(String[] args) {
                     "patient/AllergyIntolerance.read",
                     "patient/Condition.read",
                     "patient/DiagnosticReport.read",
+                    "patient/Encounter.read",
```
Comment (jblefkove-va) [inline data-query-tests/src/test/java/gov/va/api/health/dataquery/tests/dstu2/JoshBot.java:72]: JoshBot does not use the Encounter resource (it is also explicitly using dstu2 endpoints)

**Comment by jblefkove-va** [comment]
The developer portal [authorization code flow](https://developer.va.gov/explore/authorization/docs/authorization-code?api=fhir) page lives in this repo:
https://github.com/department-of-veterans-affairs/developer-portal

**Comment by taylorrassmannlibertyits** [comment]
> The developer portal [authorization code flow](https://developer.va.gov/explore/authorization/docs/authorization-code?api=fhir) page lives in this repo: https://github.com/department-of-veterans-affairs/developer-portal



> The developer portal [authorization code flow](https://developer.va.gov/explore/authorization/docs/authorization-code?api=fhir) page lives in this repo: https://github.com/department-of-veterans-affairs/developer-portal

The developer PR is now merged

**Comment by taylorrassmannlibertyits** [comment]
> The developer portal [authorization code flow](https://developer.va.gov/explore/authorization/docs/authorization-code?api=fhir) page lives in this repo: https://github.com/department-of-veterans-affairs/developer-portal



> The developer portal [authorization code flow](https://developer.va.gov/explore/authorization/docs/authorization-code?api=fhir) page lives in this repo: https://github.com/department-of-veterans-affairs/developer-portal

The developer PR is now merged

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**